pipeline-scripts/buildlib.groovy

```diff
@@ -724,6 +724,27 @@ def build_ami(major, minor, version, release, yum_base_url, ansible_branch, mail
     }
 }
 
+def sweep(build_version, sweep_builds) {
+    def sweep_job = build(
+        job: 'build%2Fsweep',
+        prapagate: false,
+        parameters: [
+            param('String', 'BUILD_VERSION', build_version),
+            param('Boolean', 'SWEEP_BUILDS', sweep_builds),
+        ]
+    )
+    if (sweep_job.result != 'SUCCESS') {
+        commonlib.email(
+            replyTo: mail_list,
```

Comment:
We always set this to `replyTo: "aos-team-art@redhat.com",` now.

Reply:
also mail_list isn't defined